portfolio/src/main/java/com/google/sps/data/TimeRange.java

```diff
@@ -30,6 +30,9 @@
   public static final Comparator<TimeRange> sortByTimeRangeStartTimeAscending =
       Comparator.comparing(TimeRange::start);
 
+  public static final Comparator<TimeRange> sortByTimeRangeDurationAscending =
```

Comment:
nit: Can you rename these constants to use CAPITAL_SNAKE_CASE?

Reply:
Updated.